drivers/gpu/drm/xen/xen_drm_zcopy_drv.c

```diff
@@ -133,23 +132,19 @@ static inline void xen_wait_obj_free(struct xen_drv_info *drv_info,
 	spin_unlock(&drv_info->wait_list_lock);
 }
 
-static void xen_wait_obj_free_all(struct xen_drv_info *drv_info)
+static inline void xen_wait_obj_check_pending(struct xen_drv_info *drv_info)
```

Comment:
Let me cite st. Torvalds:

> Often people argue that adding inline to functions that are static and used
> only once is always a win since there is no space tradeoff. While this is
> technically correct, gcc is capable of inlining these automatically without
> help, and the maintenance issue of removing the inline when a second user
> appears outweighs the potential value of the hint that tells gcc to do
> something it would have done anyway.